### ISSUE TITLE
it stops working after a few minutes

### ISSUE THREAD

**Issue by funkyn3rd** [keybase/keybase-bot#195]
I have this problem for 2-3 days.
Whatever the code, after a few minutes I am forced to restart the script because it no longer receives anything. What could be the problem? Thank you

**Comment by wiz** [comment]
Yeah, I'm also having an issue where my bot stops being connected to Keybase after a while. It seems quite random, and usually takes several hours before it gets into this state. Restarting the keybase daemon fixes it, but it's really annoying.

**Comment by WizardsOrb** [comment]
push

**Comment by pzduniak** [comment]
I'm currently working on this bug, I'll let you know if/when it gets fixed.

**Comment by malgorithms** [comment]
Once this is fixed, I recommend `init(username,paperkey)` over `initFromRunningService`.

For a bot that's designed to run for weeks/months, initializing with a username/paperkey is better because it starts its own dedicated service up that can run without interruption. In contrast, the GUI's service will shut down when you apply an update.

**Comment by al5ina5** [comment]
Hi,

I can confirm this issue also exists in my instances. Tested on both a MacBook + Ubuntu 18.04 machine. Usually lasted ~30 minutes before crashing.

I’ve mentioned this in the @mkbot channel 10+ times and most could not relate. It makes me very happy to see others are aware of the issues.

Thanks for the great work, devs!

**Comment by roychowdhuryrohit-dev** [comment]
@al5ina5 Hello is this issue fixed yet?
